lib/routes.js

```diff
@@ -214,6 +215,12 @@ module.exports = async function (app, opts) {
     notSupportedError
   } = persistedQueryProvider || {}
 
+  const normalizedRouteProps = { ...additionalRouteProps }
+  if (normalizedRouteProps.handler || normalizedRouteProps.wsHandler) {
+    normalizedRouteProps.handler = undefined
+    normalizedRouteProps.wsHandler = undefined
+  }
```

Comment:
why this change?

Reply:
The post handler adds a callback in which case fastify errors saying that there's 2 handlers. so normalised it to avoid even considering the handler provided from the options